docs(api): Robot motor control in 8.6.0 (#19262)
OpenTrons/opentrons-api: api/docs/v2/advanced_control/command_line.rst

:og:description: How to control a robot outside of the Opentrons App using the command line.

.. _command-line:

Command Line
=============

> The Python API also lets you perform many of the same actions outside of Jupyter notebook. You can control the robot by simply running commands from the command line with ``opentrons_execute``.

Reviewer: Needs a different transition here, since this starts a new page and isn't coming from the Jupyter section. 
```suggestion
The Python API also lets you perform many actions directly from the command line with ``opentrons_execute``.
```

.. TODO update with separate links to OT-2 and Flex setup, when new Flex process is in manual or on help site

> The robot's command line is accessible either by going to **New > Terminal** in Jupyter or `via SSH <https://support.opentrons.com/s/article/Connecting-to-your-OT-2-with-SSH>`_.

Reviewer: Bye bye help center link. Use https://docs.opentrons.com/flex/advanced-operation/#command-line-operation-over-ssh
